core/controllers/editor.py

```diff
@@ -303,6 +304,8 @@ def _get_exploration_data(self, exploration_id, version=None):
             'tags': exploration.tags,
             'title': exploration.title,
             'version': exploration.version,
+            'is_draft_version_valid': exp_services.is_draft_version_valid(
```

Comment:
is_draft_valid_version reads oddly to me. How about is_version_of_draft_valid, or does_draft_have_valid_version?